plugins/generator-1.16.5/forge-1.16.5/procedures/get_command_parameter_entity.java.ftl

```diff
@@ -0,0 +1,10 @@
+(new Object() {
+            public Entity getEntity() {
+                try {
+                    return EntityArgument.getEntity(arguments, "${field$param}");
+                } catch (CommandSyntaxException e) {
+                    e.printStackTrace();
+                    return null;

```

Comment:
It seems entity argument can be null, so add NULL marker to the UI of the procedure editor for this block